src/api_recievers/influxqueries.js

```diff
@@ -308,4 +308,50 @@ const grabAllPreviousStudySessionIDs = () => {
 };
 
 
-module.exports = { appData, SpecificStudySessionProcessing, grabTimesForApp, insertStudySessionData, grabAllPreviousStudySessionIDs, connectToInflux };
\ No newline at end of file
+const deleteStudySession = async (sessionId) => {
+  const deleteAPI = new DeleteAPI(influxClient);
+
+  const start = '1970-01-01T00:00:00Z'; // Start of the range (earliest possible time)
+  const stop = new Date().toISOString(); // End of the range (now)
+  const deleteKey = `_measurement="AppChange" AND QuerySession == ${sessionId}`;
+  let prevSessionDeleted = null; // Initialize outside the loop
+  
+  const headers = {
+    'Authorization': process.env.INFLUXDB_TOKEN,
+    'Content-type': 'application/json'
+  };
+
+  try {
+    await Promise.all(
+      Object.values(influxBuckets).map(async (bucket) => {
+        await deleteAPI.postDelete({
+          org,
+          bucket,
+          body: {
+            start,
+            stop,
+            deleteKey
+          },
+          headers
+        });
+
+        prevSessionDeleted = sessionId;
+
+        console.log(`You have just deleted the session associated with the ID ${sessionId}`);
+
```

Comment:
Plz remember to use our logger instead of the console logger, it's important bc the logs it generates can be read with the built version of the app

Reply:
Yup,  was just using it for debugging purposes and I forgot to remove it.